python/lsst/ts/scheduler/utils/csc_utils.py

```diff
@@ -102,3 +104,20 @@ def is_valid_efd_query(entry: str) -> bool:
         True if it is a valid EFD query.
     """
     return efd_query_re.match(entry) is not None
+
+
+def support_command(command_name: str) -> bool:
+    """Check if CSC supports a particular command.
+
+    This is used to provide backward compatibility for new commands being added
+    to the CSC.
+
+    Returns
+    -------
+    `bool`
+        True if CSC interfaces defines the command "interruptWEP", False
```

Comment:
`CSC` -> `the CSC` and either `interfaces` -> `interface` or `defines` -> `define`